firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/NotificationFeedbackTrigger.kt

```diff
@@ -221,9 +221,9 @@ class TakeScreenshotAndTriggerFeedbackActivity : Activity() {
       activity.openFileOutput(SCREENSHOT_FILE_NAME, Context.MODE_PRIVATE).use { outputStream ->
         bitmap.compress(Bitmap.CompressFormat.PNG, /* quality = */ 100, outputStream)
       }
```

Comment:
Silly GitHub doesn't let me comment on line 213... Anyhow, should there be a `finish()` at the end of `onResume()`?

Reply:
True. Out of curiosity - any reason you split the behavior between onCreate and onResume? Or could it all go in the onCreate like it is in the SDK?